.github/workflows/cypress.yml

```diff
@@ -15,12 +16,30 @@ jobs:
           sudo systemctl enable snapd
           sudo snap install maas --channel=latest/edge
           sudo snap install maas-test-db --channel=latest/edge
-      - name: Initialise MAAS
-        run: sudo maas init region+rack --maas-url=${{env.MAAS_URL}}/MAAS --database-uri maas-test-db:///
       - name: Install Cypress
         uses: cypress-io/github-action@v4
         with:
           runTests: false
+      - name: Fetch database dump
+        uses: wei/wget@v1
+        with:
+          args: -O maasdb.dump https://github.com/canonical/maas-ui-testing/raw/main/db/maasdb-22.04-master-100.dump
+      - name: Set up MAAS with database dump
+        run: |
+          sudo sed -i "s/dynamic_shared_memory_type = posix/dynamic_shared_memory_type = sysv/" /var/snap/maas-test-db/common/postgres/data/postgresql.conf
+          sudo snap restart maas-test-db
+          sudo mv maasdb.dump /var/snap/maas-test-db/common/maasdb.dump
+          sudo snap run --shell maas-test-db.psql -c 'db-dump restore /var/snap/maas-test-db/common/maasdb.dump maassampledata'
+          sudo maas init region+rack --maas-url=http://${{env.MAAS_DOMAIN}}:5240/MAAS --database-uri maas-test-db:///
+          sudo sed -i "s/database_name: maasdb/database_name: maassampledata/" /var/snap/maas/current/regiond.conf
```

Comment:
why are you giving a name of 'maassampledata' to the `db-dump restore` and then changing the configuration to point to that name, when you could `db-dump restore ... maasdb` and avoid the sed?

Reply:
Now that I know it can be done this way I'll make sure to update this, thanks!

Reply:
That seems to be throwing an error that this db already exists: `createdb: error: database creation failed: ERROR:  database "maasdb" already exists`